src/pages/SessionCreator.tsx

```diff
@@ -2,8 +2,10 @@ import Stepper from "@/components/Stepper";
 import StudentDetails from "@/components/Steps/StudentDetails";
 import { TestDetails } from "@/components/Steps/TestDetails";
 import Timeline from "@/components/Steps/Timeline";
+// import { SessionData } from "@/types/FormTypes";
```

Comment:
whats this

Reply:
resolved , was experimenting on getting session-data before